torchaudio/prototype/ctc_decoder/ctc_decoder.py

```diff
@@ -20,6 +23,15 @@
 __all__ = ["Hypothesis", "LexiconDecoder", "lexicon_decoder"]
 
 
+_PretrainedFiles = namedtuple("PretrainedFiles", ["lexicon", "tokens", "lm"])
+
+MODEL_FILE_MAP = {
```

Comment:
1. It is kind of wasteful to instantiate this mapping on module scope, so let's move this inside of `_get_filenames` function.

2. Looking ash the current mapping, the key is not encoded in the values. I see that on L305, the key (`model` variable) is still used to construct the final URL. That part should be consolidated to `_get_filenames`. So that `_get_filenames` function is responsible for providing the all the information to find the data to be downloaded, and the rest of `download_pretrained_files` function only focus on downloading and managing file on local file system.